homeassistant/components/raspberry_pi/manifest.json

```diff
@@ -5,5 +5,6 @@
   "documentation": "https://www.home-assistant.io/integrations/raspberry_pi",
   "dependencies": ["hardware", "hassio"],
   "codeowners": ["@home-assistant/core"],
-  "integration_type": "hardware"
+  "integration_type": "hardware",
+  "quality_scale": "internal"
```

Comment:
I'm not sure how this is internal?

Reply:
Same as above, no external dependencies and the documentation link is broken.